.github/ISSUE_TEMPLATE/Team_handbook.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Team handbook
+about: Propose a change to the team's handbook.
+
+---
+
+- [ ] The issue's title has a prefix of `[Team handbook]`.
+- [ ] The issue has been labeled with `Team handbook`.
+- [ ] There is more than one variant in the proposal.
```

Comment:
For new concepts this makes sense.
For changes / modifications to existing concepts, perhaps less so.

Reply:
Dropped the multiple variants requirement.